src/main/java/de/rwth/idsg/steve/config/SecurityConfiguration.java

```diff
@@ -61,11 +60,31 @@
  * @since 07.01.2015
  */
 @Slf4j
+@RequiredArgsConstructor
 @Configuration
 @EnableWebSecurity
 @Conditional(SteveProdCondition.class)
 public class SecurityConfiguration {
 
+    private final UserDetailsManager userDetailsManager;
+
+    @PostConstruct
+    public void postConstruct() {
+        String userName = SteveConfiguration.CONFIG.getAuth().getUserName();
```

Comment:
If not yet working, why not make it optional and use a default value?

Reply:
"not working" was referring to my first commit from yesterday. current state is fully working.

it even includes the optional migration of user/pwd from properties file to database. optional, because it only creates this user if the user with the same name does not exist, and hence it will be executed only for the first ever execution of this new logic.

Reply:
Sorry, it was just a global idea, not an issue for that current PR.

I recommend having a default value with "convention over configuration" in mind.
I don't remember if it already works like that or not. I think it is not.

Reply:
Forget it: `getAuth().getUserName()` should always return something (and manage a default value if needed)